### PR TITLE
Prepare Release 0.4.6

### PR DESCRIPTION
**Pull request by rafaeling**
Not many changes compared to the last release 0.4.5:
* https://github.com/eclipse-kuksa/kuksa-databroker/pull/35
* https://github.com/eclipse-kuksa/kuksa-databroker/pull/26

### CODE REVIEW

**Comment by erikbosch** [comment]
@SebastianSchildt - Databroker fails due to a license change. I have informed @rafaeling on how to update https://github.com/eclipse-kuksa/kuksa-common/tree/main/sbom-tools

But I was thinking - pointing to a specific commit makes it cumbersome to update reference in all repos affected. Would it be an idea in kuksa-common to have a tag process similar to what we have for kuksa-actions. I.e. for a minor change like this increase minor version but at the same time just move the major tag. I.e. assuming that current version is 1.0 do the change, tag it as `1.1`and at the same time move the `1` tag to point at `1.1`

**Comment by SebastianSchildt** [comment]
@erikbosch : yes probably tagging makes sense

Interesting that something changed here. Is it new sub-dependency or realy changed? I assume from Cargo lock? Dash still happy, the only findings are "normal" version bumps it seems

```
[main] INFO Querying Eclipse Foundation for license data for 163 items.
[main] INFO Found 51 items.
[main] INFO Querying ClearlyDefined for license data for 112 items.
[main] INFO Found 112 items.
[main] INFO License information could not be automatically verified for the following content:
[main] INFO 
[main] INFO crate/cratesio/-/httparse/1.9.3
[main] INFO crate/cratesio/-/regex-automata/0.4.7
[main] INFO crate/cratesio/-/regex/1.10.5
[main] INFO 
[main] INFO This content is either not correctly mapped by the system, or requires review.
```

**Comment by SebastianSchildt** [comment]
Ah checked wrong worklfow, dash also has more

```
[main] INFO Querying Eclipse Foundation for license data for 218 items.
[main] INFO Found 54 items.
[main] INFO Querying ClearlyDefined for license data for 164 items.
[main] INFO Found 164 items.
[main] INFO License information could not be automatically verified for the following content:
[main] INFO 
[main] INFO crate/cratesio/-/httparse/1.9.3
[main] INFO crate/cratesio/-/icu_collections/1.5.0
[main] INFO crate/cratesio/-/icu_locid/1.5.0
[main] INFO crate/cratesio/-/icu_locid_transform/1.5.0
[main] INFO crate/cratesio/-/icu_locid_transform_data/1.5.0
[main] INFO crate/cratesio/-/icu_normalizer/1.5.0
[main] INFO crate/cratesio/-/icu_normalizer_data/1.5.0
[main] INFO crate/cratesio/-/icu_properties/1.5.0
[main] INFO crate/cratesio/-/icu_properties_data/1.5.0
[main] INFO crate/cratesio/-/icu_provider/1.5.0
[main] INFO crate/cratesio/-/icu_provider_macros/1.5.0
[main] INFO crate/cratesio/-/idna/1.0.0
[main] INFO crate/cratesio/-/litemap/0.7.3
[main] INFO crate/cratesio/-/regex-automata/0.4.7
[main] INFO crate/cratesio/-/regex/1.10.5
[main] INFO crate/cratesio/-/tinystr/0.7.6
[main] INFO crate/cratesio/-/utf16_iter/1.0.5
[main] INFO crate/cratesio/-/utf8_iter/1.0.4
[main] INFO crate/cratesio/-/write16/1.0.0
[main] INFO crate/cratesio/-/writeable/0.5.5
[main] INFO crate/cratesio/-/yoke-derive/0.7.4
[main] INFO crate/cratesio/-/yoke/0.7.4
[main] INFO crate/cratesio/-/zerofrom-derive/0.1.4
[main] INFO crate/cratesio/-/zerofrom/0.1.4
[main] INFO crate/cratesio/-/zerovec-derive/0.10.2
[main] INFO crate/cratesio/-/zerovec/0.10.2
[main] INFO 
[main] INFO This content is either not correctly mapped by the system, or requires review.
```

**Comment by SebastianSchildt** [comment]
https://github.com/eclipse-kuksa/kuksa-common/pull/5 has been merged

Leave it up to you and @erikbosch how to update tags/CI to use new sbom-tool version

File: .github/workflows/kuksa_databroker_build.yml
Comment (erikbosch) [inline .github/workflows/kuksa_databroker_build.yml:130]: I pushed some tags (`1` and `1.0`) to kuksa-common so the line below should work as well. The advantage with that is that if we do a similar smaller update in kuksa-common (and update `1` and pushes `1.1`) we would not need to change reference here. I described the approach here: https://github.com/eclipse-kuksa/kuksa-common/wiki/Release-Process

But with that said, specifying specific commit should work as well

`pip install "git+https://github.com/eclipse-kuksa/kuksa-common.git@1#subdirectory=sbom-tools"`

**Review by erikbosch — APPROVED** [review]
Looks OK. I do not really have any opinion on if you we should use tag or ref for kuksa-common. I am happy to merge if you do no want to do any more changes @rafaeling

**Comment by SebastianSchildt** [comment]
Just a friendly reminder, after you merge this to master, some new dash tickets will be created. Pleae wait with (draft) release until no open tickets in

https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/?sort=created_date&state=opened&author_username=Kuksa-bot&first_page_size=20

(No open tickets is always safe choice. Currently there are some pending from kuksa-python-sdk fixes, so „no more triggered by databroker“ would also be fine. But anyway. None is best)

**Review by erikbosch — APPROVED** [review]

**Pull request merged**
